### PR TITLE
osquery 1.6.0

### PR DESCRIPTION
**Pull request by theopolis**
Such version bump

### CODE REVIEW

File: Library/Formula/osquery.rb [hunk 1]
```diff
@@ -3,9 +3,7 @@ class Osquery < Formula
   homepage "https://osquery.io"
   # pull from git tag to get submodules
   url "https://github.com/facebook/osquery.git",
-      :tag => "1.5.3",
-      :revision => "2a71162b0cfb040a3606538857f512ea8894b9e7"
-  revision 2
+    :tag => "1.6.0"
```
Comment (DomT4) [inline Library/Formula/osquery.rb]: Please keep the `:revision` commit reference. It acts as both a checksum and a way we can be sure the upstream tag doesn't silently wander off.

**Comment by DomT4** [comment]
Failure to compile:

```
/tmp/osquery20151104-6570-1846lbo/osquery/filesystem/filesystem.cpp:53:7: error: use of undeclared identifier 'chmod'
  if (chmod(path.c_str(), permissions) != 0) {
      ^
/tmp/osquery20151104-6570-1846lbo/osquery/filesystem/filesystem.cpp:99:15: error: variable has incomplete type 'struct stat'
  struct stat file;
              ^
/usr/include/glob.h:50:8: note: forward declaration of 'stat'
struct stat;
       ^
/tmp/osquery20151104-6570-1846lbo/osquery/filesystem/filesystem.cpp:331:15: error: variable has incomplete type 'struct stat'
  struct stat file_stat, link_stat, dir_stat;
              ^
/usr/include/glob.h:50:8: note: forward declaration of 'stat'
struct stat;
       ^
/tmp/osquery20151104-6570-1846lbo/osquery/filesystem/filesystem.cpp:331:26: error: variable has incomplete type 'struct stat'
  struct stat file_stat, link_stat, dir_stat;
                         ^
/usr/include/glob.h:50:8: note: forward declaration of 'stat'
struct stat;
       ^
/tmp/osquery20151104-6570-1846lbo/osquery/filesystem/filesystem.cpp:331:37: error: variable has incomplete type 'struct stat'
  struct stat file_stat, link_stat, dir_stat;
                                    ^
/usr/include/glob.h:50:8: note: forward declaration of 'stat'
struct stat;
       ^
```

**Comment by theopolis** [comment]
bump!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**